### ISSUE TITLE
.save method with Plotly does not work. Neither with inline or CDN resources

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#1844]
bokeh: 2.2.3, panel: 0.10.2, plotly: 4.14.1, Chrome, Windows.

-----------

On Discourse Maria asks [How to save a panel to html with plotly backend?](https://discourse.holoviz.org/t/how-to-save-a-panel-to-html-with-plotly-backend/1539).

I've tried to help but cannot get it working either.

#### With Inline Resources

```python
import panel as pn
from plotly import graph_objects as go


def _imps_device():
    fig = go.Figure(
        data=[
            go.Pie(
                labels=['a','b'],
                values=[0.3, 0.7],
            )
        ]
    )
    fig.update_layout(
        width=350,
        height=250,
        margin=dict(l=10, r=10, t=10, b=10, pad=2),
    )
    return pn.Column(
        pn.Row(pn.pane.Markdown("##Device Impressions")),
        pn.Row(
            pn.pane.Plotly(fig, config={"responsive": True}), css_classes=["center"]
        ),
    )

app = _imps_device()
from bokeh.resources import INLINE, CDN
app.save("plotly.html", resources=INLINE)
```

![image](https://user-images.githubusercontent.com/42288570/102310035-7495f500-3f6a-11eb-9d14-8387c5ef7f0a.png)

```bash
plotly.html:26 GET http://127.0.0.1:5500/static/extensions/panel/bundled/plotlyplot/plotly-latest.min.js net::ERR_ABORTED 404 (Not Found
```

#### With CDN Resources

```python
import panel as pn
from plotly import graph_objects as go


def _imps_device():
    fig = go.Figure(
        data=[
            go.Pie(
                labels=['a','b'],
                values=[0.3, 0.7],
            )
        ]
    )
    fig.update_layout(
        width=350,
        height=250,
        margin=dict(l=10, r=10, t=10, b=10, pad=2),
    )
    return pn.Column(
        pn.Row(pn.pane.Markdown("##Device Impressions")),
        pn.Row(
            pn.pane.Plotly(fig, config={"responsive": True}), css_classes=["center"]
        ),
    )

app = _imps_device()
from bokeh.resources import INLINE, CDN
app.save("plotly.html", resources=CDN)
```

![image](https://user-images.githubusercontent.com/42288570/102310118-9d1def00-3f6a-11eb-90db-064eacc1e4c6.png)

**Comment by philippjfr** [comment]
Thanks, major bug. Will want to release 0.10.3 with a fix asap.

**Comment by philippjfr** [comment]
Can't say when this was fixed but it definitely works now.

**Comment by JuliaS92** [comment]
I just tested the versions for this and seems it was fixed in 0.11.2.
